docs/CRSF.md

```diff
@@ -29,11 +29,13 @@ In EdgeTX/OpenTX, navigate to MDL->MODEL SETUP and configure the external RF mod
 Set the following parameters using the CLI or Lua script:
 
 - Tx Ch Source = crsf
-- Tx Ser Baudrate = 115200
+- Tx Ser Baudrate:
+    - 115200 for 19 Hz, 31 Hz, 50 Hz
+    - 230400 for FLRC
 - Tx Ser Dest = serial or serial2 (not mbridge!)
 - Tx Snd RadioStat = off
 
-Note: The choice of Tx Ser Baudrate is not critical and really determined by the user's need. It is recommended to set it to 57600 or higher, as this will provide enough speed for all operation modes (19 Hz, 31 Hz, 50 Hz).
+Note: Tx Ser Baudrate needs to be greater than or equal to Rx Ser Baudrate in order to provide enough capacity for the serial link.
```

Comment:
this is not exactly correct, tx and rx baudrate shoudl be larger than the link dara rate, but otherwise there is no obvious constraint.
Maybe

Note: Tx Ser Baudrate should be larger than the link data rate in order to provide enough capacity (e.g. in the 50 Hz mode the link data rate is 4100 Bytes/sec and the baudrate should thus be larger than 41000), but otherwise the choice is not critical and largely determined by the user's need. The default value is 115200, which is appropriate for all LoRa modes (50 Hz, 31 Hz, 19 Hz).

Reply:
Updated.